Configuration/ProcessModifiers/python/enableSonicTriton_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+enableSonicTriton = cms.Modifier()
```

Comment:
is this meant to enable only the FACILE variant or really everything that can potentially be using SonicTriton?
I'm trying to understand the expected design of the modifier mix.
IIUC, there will/can be many inference variants/models using SonicTriton
and many of them will be for tests only. There should be a way to distinguish.

I guess one option is to assume that this specific modifier will flip between one "default" variant of algorithms to another equivalent variant of the algorithms but remote-offloaded via SONIC+Triton. 
This option would apply to #32048: in a mix with `mlpf` modifier enabled, addition of `enableSonicTriton` will switch the way how an equivalent algorithm is executed (sort of a switch-producer).
This is clearly not the case in this PR: a completely different algorithm is substituted.

It seems to me that we need a more targeted named modifier, smth like run3_HB_FACILE or something similarly compact.

Reply:
Indeed, `enableSonicTriton` is intended to enable all available producers that use Sonic+Triton.

Introducing a separate modifier specific to this PR seems reasonable to me.

(I don't think we necessarily need to have non-offloaded versions of every ML producer, but again, I think this can be left to developers' discretion right now.)